src/main.rs

```diff
@@ -64,30 +64,39 @@ async fn main() {
             await_ready(authority.clone(), backoff).await;
 
             if shutdown {
-                // If shutdown is configured, fork the process and proxy SIGTERM.
-                let ex = fork_with_sigterm(cmd, args).await;
+                let ex = if let Some(c) = cmd {
+                    // If shutdown is configured, fork the process and proxy
+                    // SIGTERM.
+                    Some(fork_with_sigterm(c, args).await)
+                } else {
+                    None
+                };
```

Comment:
I don't think we want to support `--shutdown` without a command -- that would just terminate the proxy immediately after becoming ready.  WDYT about https://github.com/olix0r/linkerd-await/commit/01239050ed50d51ee09f4b3f5ec42e35e9e2cbd1

Reply:
Yep makes sense. I think we can use `structopt` to require `CMD` if `-S` is passed, but otherwise this makes sense.